File: editor/en/[58]Length of Last Word.py
# leetcode submit region begin(Prohibit modification and deletion)
class Solution:
    def lengthOfLastWord(self, s: str) -> int:
        s = s.strip()
        count = 0

        for i in range(len(s) - 1, -1, -1):
            if s[i] == ' ':
                return count
            count += 1

        return count

# leetcode submit region end(Prohibit modification and deletion)

File: editor/en/[58]Length of Last Word.java
//leetcode submit region begin(Prohibit modification and deletion)
class Solution {
    public int lengthOfLastWord(String s) {
        s = s.trim();
        int count = 0;

        for (int i = s.length() - 1; i >= 0; i--) {
            if (s.charAt(i) == ' ') {
                return count;
            }

            count++;
        }

        return count;
    }
}
//leetcode submit region end(Prohibit modification and deletion)
